src/components/messenger/list/index.tsx

```diff
@@ -162,7 +162,7 @@ export class Container extends React.Component<Properties, State> {
   }
 
   get isErrorDialogOpen(): boolean {
-    return this.props.isConversationErrorDialogOpen;
+    return this.props.joinRoomErrorContent !== null;
```

Comment:
```suggestion
    return !!this.props.joinRoomErrorContent;
```
Might be better because it's possible it accidentally gets set to undefined?